client/webserver/site/src/css/market.scss

```diff
@@ -288,6 +288,16 @@ table.balance-table button:hover {
   }
 }
 
+#rightSide {
+  width: calc(100% - 175px);
```

Comment:
It seems like `flex-grow: 1` would be better, but that doesn't seem to work. Hmm.

Good like this too.